### DISCUSSION TITLE
Thanks for this!

### DISCUSSION THREAD

**Discussion by johnmathews** [WhoIsSethDaniel/toggle-lsp-diagnostics.nvim#4] [General]
just wanted to say thanks for a helpful plugin. 👍

**Comment by WhoIsSethDaniel** [comment]
You're welcome. I'm glad it's helping someone (besides me).

**Comment by WhoIsSethDaniel** [comment]
If you don't mind answering: what version of neovim are you using?

**Comment by johnmathews** [comment]
0.6

**Reply by WhoIsSethDaniel** [reply]
Thanks.
